modules/gateway/peersmanager.go

```diff
@@ -30,15 +30,20 @@ func (g *Gateway) managedPeerManagerConnect(addr modules.NetAddress) {
 			// race condition could mean that the peer was disconnected
 			// before this code block was reached.
 			p.Inbound = false
-			g.log.Debugln("[PMC] [SUCCESS] existing peer has been converted to outbound peer:", addr)
-		} else {
-			g.log.Debugln("[PMC] errPeerExists was returned, but by the time the peer was to be set to 'outbound', it no longer existed.")
+			g.log.Debugf("[PMC] [SUCCESS] [%v] existing peer has been converted to outbound peer", addr)
```

Comment:
we should be more consistent with the use of "peer" vs "node". A node is an address that you know about. A peer is a node that you have connected to. This log statement could simply read "connected to node".